configs/test.yaml

```diff
@@ -1,33 +1,37 @@
 # @package _global_
 
-# specify here default evaluation configuration
 defaults:
   - _self_
   - dataset: conll2003
   - datamodule: default
   - callbacks: null
   - logger: null
   - trainer: default.yaml
-  - log_dir: evaluation.yaml
-
+  - hydra: default.yaml
+  - paths: default.yaml
   - experiment: null
 
   # enable color logging
   - override hydra/hydra_logging: colorlog
   - override hydra/job_logging: colorlog
```

Comment:
as above, is this already defined in `hydra/defaults.yaml`, isn't it?

Reply:
fixed in [e884f23](https://github.com/ChristophAlt/pytorch-ie-hydra-template/pull/42/commits/e884f233ad799e3bbebd68b922e6570403f7b773)